scripts/deploy_master.sh

```diff
@@ -22,8 +22,11 @@ PR_NUM=`git log --format=%B -n 1 $SHA | grep -Eo '#[0-9]+' | tail -n 1 | cut -d
 # format hardcoded because https://stackoverflow.com/questions/7216358/date-command-on-os-x-doesnt-have-iso-8601-i-option
 DATE=`date -u +"%Y-%m-%dT%H:%M:%SZ"`
 
+# Pulls the native code version from the package.json
+NATIVE_VERSION=`python -c "import json;print json.loads(open('package.json', 'r').read())['native-code-version']"`
```

Comment:
🐍????

We already use Ruby in this repo, can we limit to that?

Reply:
alright, alright

Reply:
its now ruby 🍡